src/medCore/gui/viewContainers/medViewContainer.cpp

```diff
@@ -331,25 +336,29 @@ void medViewContainer::setUserSplittable(bool splittable)
 {
     d->userSplittable = splittable;
 
-    medAbstractImageView *view = dynamic_cast <medAbstractImageView *> (d->view);
-
     if(d->userSplittable)
     {
-          d->hSplitAction->setEnabled(true);
-          d->vSplitAction->setEnabled(true);
-          d->presetMenu->setEnabled(true);
+        d->hSplitAction->setEnabled(true);
+        d->vSplitAction->setEnabled(true);
+        d->presetMenu->setEnabled(true);
 
+        medAbstractImageView *view = dynamic_cast <medAbstractImageView *> (d->view);
         if (view)
-            view->fourViewsParameter()->show();
+        {
+            d->histogramAction->setEnabled(true);
+            d->fourSplitAction->setEnabled(true);
+        }
+
+        d->maximizedAction->setEnabled(true);
     }
     else
     {
         d->hSplitAction->setEnabled(false);
         d->vSplitAction->setEnabled(false);
         d->presetMenu->setEnabled(false);
-
-        if (view)
-            view->fourViewsParameter()->hide();
+        d->fourSplitAction->setEnabled(false);
+        d->maximizedAction->setEnabled(false);
+        d->histogramAction->setEnabled(false);
```

Comment:
The splitting option is more based on the workspace acceptance/usability of several views. The histogram is not a view.